sources/packages/backend/apps/api/src/route-controllers/student/student.students.controller.ts

```diff
@@ -280,13 +280,17 @@ export class StudentStudentsController extends BaseController {
     // This method will be executed alongside with the transaction during the
     // execution of the method updateStudentFiles.
     const sendFileUploadNotification = () =>
-      this.gcNotifyActionsService.sendFileUploadNotification({
-        firstName: student.user.firstName,
-        lastName: student.user.lastName,
-        birthDate: student.birthDate,
-        documentPurpose: payload.submittedForm.documentPurpose,
-        applicationNumber: payload.submittedForm.applicationNumber,
-      });
+      this.gcNotifyActionsService.sendFileUploadNotification(
```

Comment:
Previously this method was executing just an API call to GC Notify but now it is also saving to DB. I believe that we need to allow the notification to be saved to the DB in the same transaction that is updated in the uploaded file to the DB.
This would be the same behavior for all the notifications.

Reply:
I left this as discussed yesterday and this morning due to the current issue I encountered after refactoring the code to use entity manager.

Reply:
@ann-aot @dheepak-aot @guru-aot @andrepestana-aot this will be part of the next ticket. We can discuss this in the next grooming.

Reply:
Sure @andrewsignori-aot. we can discuss in next grooming.